index.html

```diff
@@ -0,0 +1,641 @@
+        </p>
+        <br>
+        <p>
+          If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and irregularities. Those particles were not missing, but would be diluted beyond detection, like spit in the ocean.
+        </p>
+        <br>
+        <p>
+          “SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.
+        </p>
+        <br>
+        <p>
+          On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.
+        </p>
+        <br>
+        <p>
+          Reaching back across 13.8 billion years to <a class="article-links" href="https://www.youtube.com/watch?v=ZJYc9YmKIO8&amp;feature=youtu.be">the first sliver of cosmic time</a> with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of space-time — so-called <a class="article-links" href="http://www.nytimes.com/2016/02/12/science/ligo-gravitational-waves-black-holes-einstein.html">gravitational waves</a> — the signature of a universe being wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth of a second old. They are the long-sought smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.
+        </p>
+        <hr class="nytline">
+      </div>  
+    </div>
+    <div id="floatleft">
+      <p id="hidden-paragraph">lorem ipsum Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</p>
+    </div>  
+    <div id="thwrapper">
+      <div id="thcont">
+        <div id="thanks" class="center-items">          
+          <p class="bold">Thanks for reading The Times.</p>
+          <a href="https://www.nytimes.com/subscription?campaignId=7UU8U" class="bold">Subscribe to The Times</a>         
+        </div>
+      </div>
+    </div>
+    <aside id="asidebox">
+      <div id="asideboxmt">
+        <div id="asideitems">
```

Comment:
please changes the id's to classes you are using them a lot, it's not a good practice at all.